.github/workflows/publish.yml

```diff
@@ -9,8 +9,8 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: subosito/flutter-action@v1
+      - uses: actions/checkout@v3
+      - uses: subosito/flutter-action@v2
         with:
           channel: stable
```

Comment:
if you want to be a bit more safe for future funny stable releases you could use the

```
env:
  FLUTTER_VERSION: '3.10.x'
```
trick and then in each job
```
 with:
          channel: 'stable'
          flutter-version: ${{env.FLUTTER_VERSION}}
          ```